components/server/src/workspace/workspace-starter.ts

```diff
@@ -307,7 +307,10 @@ export class WorkspaceStarter {
                     ideSettings = {
                         ...ideSettings,
                         defaultIde: ideConfig.ide,
-                        useLatestVersion: !!ideConfig.useLatest,
+                        useLatestVersion:
+                            ideSettings?.useLatestVersion ??
+                            user.additionalData?.ideSettings?.useLatestVersion ??
+                            !!ideConfig.useLatest,
```

Comment:
BTW, are we fixing something here? It looks like we're changing the prioritization of what gets to dictate whether the latest version of the editor is used, so I'm curious why it's needed 👀

Reply:
See https://linear.app/gitpod/issue/ENT-455

Reply:
Oh nice, thanks!